etcdctl/command/cluster_health.go

```diff
@@ -46,9 +46,10 @@ func handleClusterHealth(c *cli.Context) {
 	}
 
 	// do we have a leader?
-	ep, ls0, err := getLeaderStats(tr, client.GetCluster())
+	cl := client.GetCluster()
+	ep, ls0, err := getLeaderStats(tr, cl)
 	if err != nil {
-		fmt.Println("cluster is unhealthy")
+		fmt.Println("failed to get leader stats from cluster urls", cl)
```

Comment:
fmt.Println("cannot determine the health of the cluster: fail to connect to the leader")